packages/app/src/components/pages/Home/useSort.js

```diff
@@ -28,14 +28,14 @@ const useSort = () => {
       let aDate = new Date(a[sortConfig.key])
       let bDate = new Date(b[sortConfig.key])
 
-      if (aDate > bDate) {
-        return sortConfig.direction === 'ascending' ? -1 : 1
-      }
-
       if (aDate < bDate) {
         return sortConfig.direction === 'ascending' ? 1 : -1
       }
 
+      if (aDate > bDate) {
+        return sortConfig.direction === 'ascending' ? -1 : 1
```

Comment:
It would be nice to switch all of these 'ascending' and 'descending' strings to use an enum, that way there's less chance of a typo causing a bug.

Reply:
Created enum for this functionality.